picky-server/src/http/controller.rs

```diff
@@ -229,6 +232,88 @@ impl ServerController {
             }
         }
     }
+
+    #[post("/timestamp")]
+    async fn authenticode_timestamp(&self, req: Request) -> Result<ResponseBuilder, StatusCode> {
+        let req = req.load_body().await.bad_request()?;
+
+        let mut body = req.body().to_vec();
+        body.retain(|&x| x != 0x0d && x != 0x0a && x != 0x00); // do not include CRL
```

Comment:
I’m not sure what you’re doing here

Reply:
Removing CRLF entries

Reply:
Oh right
For clarity I suggest using `b'\n'`, `b'\r'` and `b'\0'` instead of the hexadecimal value and update the comment because currently it states `CRL` instead of `CRLF`